torch/package/package_exporter.py

```diff
@@ -511,6 +514,7 @@ def _intern_module(
                 is_package=is_package,
                 error=packaging_error,
                 error_context=error_context,
+                provided=True,
```

Comment:
I'm guessing this is necessary to prevent `add_dependency` from adding another node for this module in the dependency graph, but it is counterintuitive for this to be "provided" since there is a packaging error attached. The graph will fail validation and won't be executed.

Reply:
"provided" just means "don't process it again" (probably should have a better name). Previously this was added in `save_module` above but I deleted that line so I had to add it back here.